mhs-reference-implementation/mhs/config/interactions.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+class InteractionsFile:
+    def __init__(self, interactions_file_name):
+        """Create a new InteractionsFile that loads the interaction details from the specified JSON file.
+
+        :param interactions_file_name: The file to load interaction details from. This file should be in JSON format.
+        """
+        with open(interactions_file_name) as interactions_file:
+            self.interactions = json.loads(interactions_file.read())
```

Comment:
Not really worth changing but here you can use `json.load(interactions_file)` to get rid of the `read()` call. 

What might be worth doing though is moving this to the utility class if we keep having to load json into a dictionary, I know I've written this code at least once before in SCR.

Reply:
That's a good call (on both points). Since it'll be such a small change, I'll move it into the common module as part of this PR.